### PR TITLE
hearing Serv auth Token impl

### PR DESCRIPTION
**Pull request by hmcts-shajahan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java [hunk 1]
```diff
@@ -37,6 +39,8 @@ public class HearingsController {
     public static final String PROCESSING_REQUEST_AFTER_AUTHORIZATION =
             "processing request after authorization";
 
+    public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
```
Comment (hmcts-kodatala) [inline src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java:42]: Move to constant file above two constants

**Review by hmcts-kodatala — COMMENTED** [review]
Move constants to constant file

File: src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java (hunk 1 above)
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java]: Moved

**Review by hmcts-kodatala — APPROVED** [review]
OK

**Pull request merged**